README.md

```diff
@@ -11,7 +11,17 @@ This application defines global templates for [GOV.UK](https://www.gov.uk) pages
 
 This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
-You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the app. Remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the app. Remember to combine it with the commands that follow.
+
+### Running the app
+
+Run the app [using GOV.UK Docker](https://github.com/alphagov/govuk-docker#usage). You can then view your templates - here's an example:
```

Comment:
Minor: we don't need to say this again (we said as much in the previous paragraph). You could start with e.g.

```
Static has no home page or navigation, so you need to manually type in the URL of the template you want to view. Here's an example:
```